pkg/provisioner/k8sprovisioner.go

```diff
@@ -111,21 +117,21 @@ func createPodMeta() v1.Pod {
 			Labels:       labels,
 		},
 		Spec: v1.PodSpec{
-			AutomountServiceAccountToken: new(bool),
-			RestartPolicy:                v1.RestartPolicyNever,
+			ServiceAccountName: "wharf-builder",
+			RestartPolicy:      v1.RestartPolicyNever,
 			InitContainers: []v1.Container{
 				{
 					Name:            "init",
 					Image:           "bitnami/git:2-debian-10",
 					ImagePullPolicy: v1.PullIfNotPresent,
-					Args:            append(podInitCloneArgs, "http://github.com/iver-wharf/wharf-cmd", repoVolumeMountPath),
+					Args:            append(podInitCloneArgs, "-b", "feature/aggregator-issue-15", "http://github.com/iver-wharf/wharf-cmd", repoVolumeMountPath),
 					VolumeMounts:    volumeMounts,
 				},
 			},
 			Containers: []v1.Container{
 				{
 					Name:            "app",
-					Image:           "ubuntu:20.04",
+					Image:           "golang:latest",
```

Comment:
During testing, instead of downloading and installing Go to the ubuntu image I decided to use this, should I revert back to `ubuntu:20.04`?

Reply:
Added todo comment.